packages/fs-detectors/test/unit.local-file-system-detector.test.ts

```diff
@@ -70,6 +72,31 @@ describe('LocalFileSystemDetector', () => {
     expect(actualPaths).toEqual(expectedPaths);
   });
 
+  it('should skip entry if socket', async () => {
+    // Windows does not support Unix domain sockets
+    if (platform === 'win32') {
+      return;
+    }
```

Comment:
```suggestion
  // Skipping on Windows because it does not support Unix domain sockets
  it.skipIf(process.platform === 'win32')('should skip entry if socket', async () => {
```